templates/dashboard/alias_transfer_send.html

```diff
@@ -29,18 +25,40 @@ <h1 class="h3">Transfer {{ alias.email }}</h1>
           {{ alias_transfer_url }}
         </em>
 
+        <p class="mt-5">
+            Please copy the transfer URL. <strong>We won't be able to display it again</strong>. If you need to access it again you can generate a new URL.
+        </p>
+
+        <p class="mt-2">
+            This transfer URL is <strong>valid for 30 minutes</strong>. If it hasn't been used by then it will be automatically disabled.
```

Comment:
If we change the 30 minutes, we need to update the wording here too.

Reply:
changed.